File: spring/src/main/java/ua/com/online/appointment/response/WorkerInfoResponse.java
package ua.com.online.appointment.response;

import lombok.Data;

@Data
public class WorkerInfoResponse {
    private String username;
    private String fullname;
    private String email;
    private String phone;
    private String workStartsIn;
    private String workEndsIn;
    private Integer mondayFlag;
    private Integer tuesdayFlag;
    private Integer wednesdayFlag;
    private Integer thursdayFlag;
    private Integer fridayFlag;
    private Integer saturdayFlag;
    private Integer sundayFlag;
}

File: angular/src/assets/response-and-request/WorkerInfoResponse.ts
export class WorkerInfoResponse{
  username:string;
  fullname:string;
  email:string;
  phone:string;
  workStartsIn:string;
  workEndsIn:string;
  mondayFlag:any;
  tuesdayFlag:any;
  wednesdayFlag:any;
  thursdayFlag:any;
  fridayFlag:any;
  saturdayFlag:any;
  sundayFlag:any;


  constructor() {
    this.username = '';
    this.fullname = '';
    this.email = '';
    this.phone = '';
    this.workStartsIn = '';
    this.workEndsIn = '';
    this.mondayFlag = null;
    this.tuesdayFlag = null;
    this.wednesdayFlag = null;
    this.thursdayFlag = null;
    this.fridayFlag = null;
    this.saturdayFlag = null;
    this.sundayFlag = null;
  }
}
